### PR TITLE
Docs: production checklist

### PR DESCRIPTION
**Pull request by reinkrul**
Probably far from complete, please elaborate.

Fixes https://github.com/nuts-foundation/nuts-node/issues/1645

### CODE REVIEW

File: docs/pages/deployment/recommended-deployment.rst [hunk 1]
```diff
@@ -133,4 +133,22 @@ Private Key Storage
...
+  * Make sure only correct CA certificates are in truststore (depends on network)
+* Key Management
+  * Have a scheduled key rotation procedure
+* Backup Management
+  * Make sure data is backed up
+  * Have a tested backup/restore procedure
+* Configuration
+  * Make sure ``strictmode`` is enabled
+* Security
+  * Protect access to HTTP ``/internal``
```
Comment (woutslakhorst) [inline docs/pages/deployment/recommended-deployment.rst:154]: and do not expose

Comment (woutslakhorst) [reply docs/pages/deployment/recommended-deployment.rst:154]: + only expose X, the rest not

File: docs/pages/deployment/recommended-deployment.rst [hunk 2]
```diff
@@ -133,4 +133,22 @@ Private Key Storage
...
\ No newline at end of file
+Refer to the config options of the crypto engine and `Vault documentation <https://www.vaultproject.io/docs>`_ for configuring it.
+
+Production Checklist
+********************
+
+Below is a list of items that should be addressed when running a node in production:
+
+* TLS
+  * Require client certificate on HTTP ``/n2n`` and gRPC endpoints.
```
Comment (woutslakhorst) [inline docs/pages/deployment/recommended-deployment.rst:144]: ambiguous regarding reverse proxy

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: docs/pages/deployment/recommended-deployment.rst [hunk 3]
```diff
@@ -133,4 +133,24 @@ Private Key Storage
...
\ No newline at end of file
+Refer to the config options of the crypto engine and `Vault documentation <https://www.vaultproject.io/docs>`_ for configuring it.
+
+Production Checklist
+********************
+
+Below is a list of items that should be addressed when running a node in production:
+
+* TLS
+  * Use a proxy in front of the node which terminates TLS
```
Comment (stevenvegt) [inline docs/pages/deployment/recommended-deployment.rst:144]: Say something about a proxy which prevents connection spamming?

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**